.gitignore

```diff
@@ -1 +1,5 @@
 .idea
+.tool-versions
+.envrc
```

Comment:
These are just for my local environment, I can pull them from the PR and just keep them locally if they bug ya